bin.src/showDither.py

```diff
@@ -40,7 +40,7 @@ def main(butler, visits, fields, fieldRadius, showCCDs=False, aitoff=False, alph
         visits.append(v)
         ra.append(sky[0].asDegrees())
         dec.append(sky[1].asDegrees())
-        filters[v] = exp.getFilter().getName()
+        filters[v] = exp.getFilterLabel().bandLabel
 
     plt.clf()
     if aitoff:
```

Comment:
This looks fine by itself, but in the code below there's a dictionary that expects `r1` as one of the filters.

Reply:
Ugh. I was hesitant to even touch this file, since it's an executable from 2013 that probably doesn't work anyway. I'm not sure what to do here: there is no good solution and I cannot test this code. Do we know if it even works?

Reply:
Actually, that's fine: `ENG-R1` has band `r1`, so `bandLabel` is absolutely correct here.

Reply:
Yes: https://github.com/lsst/obs_subaru/blob/master/python/lsst/obs/hsc/hscFilters.py#L63-L64